### PR TITLE
[arrays_of_tuples] Various updates

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: src/Test/contconic.jl [hunk 1]
```diff
@@ -993,7 +993,7 @@ function _geomean1test(model::MOI.ModelLike, config::TestConfig, vecofvars, n=3)
     @test MOI.get(model, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}()) == 1
 
     @test MOI.canset(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}())
-    MOI.set!(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction([t], [1.], 0.))
+    MOI.set!(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([[1.]], [[t]]), 0.))
```
Comment (odow) [inline src/Test/contconic.jl]: Why the double brackets?

File: src/Test/contconic.jl [hunk 2]
```diff
@@ -145,7 +145,7 @@ function _lin2test(model::MOI.ModelLike, config::TestConfig, vecofvars::Bool)
         vz = MOI.addconstraint!(model, [z], MOI.Nonnegatives(1))
     else
         @test MOI.canaddconstraint(model, MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives)
-        vz = MOI.addconstraint!(model, MOI.VectorAffineFunction([1], [z], [1.], [0.]), MOI.Nonnegatives(1))
+        vz = MOI.addconstraint!(model, MOI.VectorAffineFunction(MOI.VectorAffineTerm.([[1]], MOI.ScalarAffineTerm.([[1.]], [[z]])), [0.]), MOI.Nonnegatives(1))
```
Comment (odow) [inline src/Test/contconic.jl]: Trailing `0`'s again

File: src/Test/contconic.jl [hunk 3]
```diff
@@ -230,9 +230,9 @@ function lin3test(model::MOI.ModelLike, config::TestConfig)
     x = MOI.addvariable!(model)
 
     @test MOI.canaddconstraint(model, MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives)
-    MOI.addconstraint!(model, MOI.VectorAffineFunction([1],[x],[1.0],[-1.0]), MOI.Nonnegatives(1))
+    MOI.addconstraint!(model, MOI.VectorAffineFunction(MOI.VectorAffineTerm.([1], MOI.ScalarAffineTerm.([1.0], [x])),[-1.0]), MOI.Nonnegatives(1))
```
Comment (mlubin) [inline src/Test/contconic.jl]: Unnecessary broadcast? Why not:
```julia
MOI.VectorAffineTerm(1, MOI.ScalarAffineTerm(1.0, x))
```

**Comment by blegat** [comment]
Everything is working again except

- parser
- UnitTests
- Bridges

whose tests have been commented.

I suggest to merge this PR into `array_of_tuples` and then fix these last three separately

File: src/Test/contconic.jl (hunk 3 above)
Comment (blegat) [reply src/Test/contconic.jl]: One other think that could be changed would be thinks like
```julia
MOI.ScalarAffineTerm.(one(3), [x, y, z])
```
into
```julia
MOI.ScalarAffineTerm.(1.0, [x, y, z])
```

**Pull request merged**